adddigits.py

```diff
@@ -0,0 +1,12 @@
+#https://leetcode.com/problems/add-digits/
```

Comment:
Can you add the actual problem as a comment here

```suggestion
Given an integer num, repeatedly add all its digits until the result has only one digit, and return it.

Input: num = 38
Output: 2
Explanation: The process is
38 --> 3 + 8 --> 11
11 --> 1 + 1 --> 2 
Since 2 has only one digit, return it.
```